data-structures/linked_list/.pytest_cache/v/cache/lastfailed

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
You need a `.gitignore` in the root of your repo. These files are being generated by python and should not be uploaded to GitHub. Make sure your `.gitignore` includes files from your OS and editor as well as python. [gitignore.io](https://www.gitignore.io/) is a great resource for making a `.gitignore`.